build_pdf4qt.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+# Install VCPKG
+git clone https://github.com/microsoft/vcpkg.git
+cd vcpkg
+./bootstrap-vcpkg.sh -disableMetrics
+cd ..
+export VCPKG_ROOT=$(pwd)/vcpkg
+
+# Install tbb
+${VCPKG_ROOT}/vcpkg install tbb
+
+# Build
+cd pdf4qt
+export CMAKE_BUILD_PARALLEL_LEVEL=$(nproc)
+cmake -B build -S . -DCMAKE_BUILD_TYPE=Release -DPDF4QT_INSTALL_QT_DEPENDENCIES=0 -DCMAKE_TOOLCHAIN_FILE=${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake -DCMAKE_INSTALL_PREFIX=/app
+cmake --build build
+cmake --install build
+
+# Move icons and desktop files to the right place
+cd ..
+
+mkdir -p /app/lib/pdf4qt
+mkdir -p /app/share/icons
+mkdir -p /app/share/applications
+mkdir -p /app/share/metainfo
+
+mv -f /app/lib/*Plugin.so*  /app/lib/pdf4qt/
+mv -f /app/usr/share/icons/*.svg /app/share/icons/
+mv -f /app/usr/share/applications/*.desktop /app/share/applications/
+mv -f .appdata.xml /app/share/metainfo/io.github.JakubMelka.Pdf4qt.appdata.xml
+
+rmdir /app/usr/share/icons
+rmdir /app/usr/share/applications
+rmdir /app/usr/share
+rmdir /app/usr
+
+# Edit the files
+
+sed -i 's|/usr/bin/||g' /app/share/applications/*.desktop
+sed -i 's|/usr/share/icons/||g' /app/share/applications/*.desktop
```

Comment:
if you reference these path is you desktop files, then the desktop files are wrong.

Reply:
Desktop files are prepared for standard environments (like creating a .deb package) for which they are functional. However, Flatpak is configured differently. I didn't want two separate versions of Desktop files (one for Flatpak, the other for standard Linux), so I resolved it by editing.

Reply:
no. that's just wrong. I maintain what I said. A .desktop file NEVER SHOULD reference an absolute path be they icons or executables.

Reply:
you also fail to honour `PREFIX`.

Reply:
It should be fixed.